.github/workflows/main.yml

```
@@ -69,112 +74,199 @@ jobs:
+          AWS_REGION: ${{ secrets.AWS_REGION }}
+          NAMESPACE: ${{ secrets.NAMESPACE }}
+          DEPLOYMENT_BRANCH: ${{ github.event.inputs.environment || 'staging' }}
+          BACKEND_IMAGE: ${{ needs.build-and-push.outputs.backend_image }}
+          FRONTEND_IMAGE: ${{ needs.build-and-push.outputs.frontend_image }}
+          PROJECT_URL: ${{ secrets.PROJECT_URL }}
+          DOCKER_SERVER: ${{ secrets.DOCKER_SERVER }}
+          AWS_EMAIL: ${{ secrets.AWS_EMAIL }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         if: success()
         run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:57: Double quote to prevent globbing and word splitting

(shellcheck)